skillmap/src/components/AppModal.tsx

```diff
@@ -90,6 +93,25 @@ export class AppModalImpl extends React.Component<AppModalProps> {
         </Modal>
     }
 
+    renderResetWarning() {
+        const  { dispatchResetUser, dispatchHideModal } = this.props;
+        const resetModalTitle = lf("Reset All Activities?");
+        const resetModalText = lf("Are you sure you want to reset? This will permanently erase all progress and delete the current projects. This action cannot be undone.");
```

Comment:
"delete the projects you created in tutorials"? struggling with how to convey that it's really just deleting the projects in the skillmap and not the main editor